src/main/scala/io/viash/config/ComputationalRequirements.scala

```diff
@@ -47,8 +47,8 @@ case class ComputationalRequirements(
     memory.map(_.toLowerCase()) match {
       case Some(Regex(amnt, unit)) => 
         val amntBigInt = BigInt(amnt)
-        val multiplier = BigInt(1024)
-        val exp = lookup(unit)
+        val multiplier = if (unit.contains("i")) BigInt(1024) else BigInt(1000)
+        val exp = lookup(unit.replace("i", ""))
```

Comment:
why not simply add `kib`, `mib`, ... to the lookup?

Reply:
Valid question! I originally had it like that and replaced it as I found the code very bulky. I'll revisit and maybe try to come up with something better.